### PR TITLE
Vijaya1

### PR DESCRIPTION
**Pull request by vijaya1811**

### CODE REVIEW

File: vijaya/html/assignment3/contact.html [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+        contact us  contact us contact us contact us contact us contact us 
+        contact us  contact us contact us contact us contact us contact us 
+        contact us  contact us contact us contact us contact us contact us 
+    </p>
+
+  
+</div>
+
+
+
```
Comment (shubhamdalal49) [inline vijaya/html/assignment3/contact.html]: Remove this extra space

File: vijaya/html/assignment3/index.html [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+<div class="container" style="display: flex;">
+        <img src="./img/purpleflower.jpg" width="400" height="300"> 
+        <p style="width: 900px; margin-left:10px;">This is home page This is home page This is home page  This is home page This is home page  
+        This is home page This is home page This is home page This is home page This is home page This is home page This is home page 
+        This is home page This is home page This is home page This is home page This is home page This is home page This is home page 
+        This is home page This is home page This is home page This is home page This is home page This is home page This is home page 
+        This is home page This is home page This is home page This is home page This is home page This is home page This is home page 
+        This is home page This is home page This is home page This is home page This is home page This is home page This is home page 
+        This is home page This is home page This is home page This is home page This is home page  
+     </p>
```
Comment (shubhamdalal49) [inline vijaya/html/assignment3/index.html]: correct indentation

File: vijaya/html/assignment3/mean.html [hunk 3]
```diff
@@ -0,0 +1,31 @@
+<html>
+<head>
+<title>Assignment 2</title>
+</head>
+<style>
+h1 {text-align: center;}
+.one{text-align: center;}
+.bg{background-color: aqua;}
+</style>
```
Comment (shubhamdalal49) [inline vijaya/html/assignment3/mean.html:8]: style tag should be in head tag. please change it and put it in head tag

File: vijaya/html/assignment3/question1.html [hunk 4]
```diff
@@ -0,0 +1,44 @@
+<html>
+<head>
+<title>Assignment 2</title>
+</head>
+<style>
+h1 {text-align: center;}
+.one{text-align: center;}
+</style>
```
Comment (shubhamdalal49) [inline vijaya/html/assignment3/question1.html:7]: Style tag should be in head tag

File: vijaya/html/assignment4/question1.html [hunk 5]
```diff
@@ -0,0 +1,45 @@
+
+<!DOCTYPE html>
+<html>
+<head>
+<style>
+table{ width:500px; height: 200px; border-collapse:collapse; text-align:center;}
+table, th, td {
+  border: 1px solid black;
+}
+</style>
```
Comment (shubhamdalal49) [inline vijaya/html/assignment4/question1.html:10]: style should be in  head tag

File: vijaya/html/assignment4/question1.html [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+<caption><em>A test table with merged cells</em></caption>
+
+<tr>
+    <th rowspan="2"></th ><th  colspan="2">Average</th>
+    <th  rowspan="2">Red <br>eyes</th>
+</tr>
+
+<tr>
+  <th >height</th> <th>weight</th>
+</tr>
```
Comment (shubhamdalal49) [inline vijaya/html/assignment4/question1.html:29]: your code should be in proper format

**Review by shubhamdalal49 — CHANGES_REQUESTED** [review]

File: vijaya/html/assignment3/contact.html [hunk 7]
```diff
@@ -0,0 +1,58 @@
+<html>
+<head>
+<title>Assignment 2</title>
+
+</head>
+<style>
+h1 {text-align: center;}
+.one{text-align: center;}
+.bg{background-color: aqua;}
+
```
Comment (nileshp1122) [inline vijaya/html/assignment3/contact.html]: remove extra space

**Review by Ashwin2812 — APPROVED** [review]

**Review by Ashwin2812 — APPROVED** [review]

**Review by nileshp1122 — APPROVED** [review]

**Review by shubhamdalal49 — APPROVED** [review]

**Review by Ashwin2812 — APPROVED** [review]

**Review by urmilagagare — APPROVED** [review]

**Pull request merged**